webextensions/api/dom.json

```diff
@@ -0,0 +1,28 @@
+{
+  "webextensions": {
+    "api": {
+      "dom": {
+        "openOrClosedShadowRoot": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Mozilla/Add-ons/WebExtensions/API/dom/openOrClosedShadowRoot",
```

Comment:
Is there a mdn/content ticket that tracks the documentation?

Reply:
This PR will act as the tracker for the document move with the `needs content update` label -- I forgot to add it before, so I've added it now!